docs/src/development/local/lando.md

```diff
@@ -1,67 +1,45 @@
 ---
-title: "Lando"
+title: Using Lando for local development
+sidebarTitle: "Lando"
 weight: 3
 ---
 
-# Using Lando for local development
+[Lando](https://github.com/lando/lando) is Platform.sh's recommended local development tool.  It is a third party tool developed by [Tandem](https://thinktandem.io), which now includes direct support for Platform.sh projects.  Lando can read your Platform.sh configuration files and produce an approximately equivalent configuration using Docker with minimal effort on the part of the developer.
```

Comment:
"approximately equivalent" might be underselling how close the configuration actually is